frontends/web/tests/send.test.ts

```diff
@@ -0,0 +1,143 @@
+/**
+*  Copyright 2025 Shift Crypto AG
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*       http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+import { expect } from '@playwright/test';
+import { test } from './helpers/fixtures';
+import { ServeWallet } from './helpers/servewallet';
+import { launchRegtest, setupRegtestWallet, sendCoins, mineBlocks, cleanupRegtest } from './helpers/regtest';
+import { ChildProcess } from 'child_process';
+import { deleteAccountsFile } from './helpers/fs';
+
+let servewallet: ServeWallet;
+let regtest: ChildProcess;
+
+test('Send BTC', async ({ page, host, frontendPort, servewalletPort }, testInfo) => {
+
+  await test.step('Start regtest and init wallet', async () => {
+    regtest = await launchRegtest();
+    // Give regtest some time to start
+    await new Promise((resolve) => setTimeout(resolve, 3000));
+    await setupRegtestWallet();
+  });
+
+
+  await test.step('Start servewallet', async () => {
+    servewallet = new ServeWallet(page, servewalletPort, frontendPort, host, testInfo.title, testInfo.project.name, { regtest: true, testnet: false });
+    await servewallet.start();
+  });
+
+
+  let recvAdd: string;
+  await test.step('Grab receive address', async () => {
+    await page.getByRole('button', { name: 'Test wallet' }).click();
+    await page.getByRole('button', { name: 'Unlock' }).click();
+    await page.getByRole('link', { name: 'Bitcoin Regtest Bitcoin' }).click();
+    await page.getByRole('button', { name: 'Receive RBTC' }).click();
+    await page.getByRole('button', { name: 'Verify address on BitBox' }).click();
+    const addressLocator = page.locator('[data-testid="receive-address"]');
+    recvAdd = await addressLocator.inputValue();
+    console.log(`Receive address: ${recvAdd}`);
+  });
+
+  await test.step('Verify there are no transactions yet', async () => {
+    await page.goto('/#/account-summary');
+    mineBlocks(12);
+    await page.locator('td[data-label="Account name"]').nth(0).click();
+    await expect(page.getByTestId('transaction')).toHaveCount(0);
+  });
+
+  await test.step('Add second RBTC account', async () => {
+    await page.goto('/#/account-summary');
+    await page.getByRole('link', { name: 'Settings' }).click();
+    await page.getByRole('link', { name: 'Manage Accounts' }).click();
+    await page.getByRole('button', { name: 'Add account' }).click();
+    await page.getByRole('button', { name: 'Add account' }).click();
+    await page.getByRole('button', { name: 'Done' }).click();
```

Comment:
At the end of adding a 2nd account there should be a text: "Account added" or "\<accountname\> has now been added to your accounts."

Maybe something to test here with `expect`?